cluster/Vagrantfile

```diff
@@ -96,7 +96,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
           c.vm.provision :shell, path: 'bootstrap_centos.sh'
           if role == 'master'
             # Install kubernetes on master
-            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.4.7'
+            ks8_ver = ENV['CONTIV_K8s_VERSION'] || 'v1.5.2'
```

Comment:
Good point, its just that kubeadm users are likely going to keep moving on faster. I will change it back to 1.4.7 for now and we can discuss version support policies etc.